content(cms): create Post "einführung-tei-übungsaufgabe/index" (#19)
acdh-oeaw/howto: content/posts/einführung-tei-übungsaufgabe/index.mdx

![Filmwelt 31, 1924, S. 11, mit Bereichsannotation](images/filmwelt_1923_31_11_annotiert.jpg)

![Filmwelt 31, 1924, S. 12, mit Bereichsannotation](images/filmwelt_1923_31_12_annotiert.jpg)

Anhand dieser Vorlage sollen beispielhaft einige Facetten der Textannotation durchgeführt werden. Neben verschiedenen Textsorten sind vielfältige Strukturen wie Überschriften, Paragrafen, Verszeilen, Kopf- und Fußzeilen sowie eine Liste vorhanden. Darüber hinaus gibt es Zeilen- und Seitenumbrüche, Fett- und Sperrdruck, Datumsangaben, Personen- und Ortsnamen, eine Abbildung und sogar einen Druckfehler.  All diese Komponenten und Texteigenschaften werden mit Hilfe der TEI vom analogen zum digitalen Format, das zahlreiche Nutzungsmöglichkeiten bietet und damit neue Perspektiven auf die Texte erlaubt.

Ein wichtiges Hilfsmittel für die Arbeit mit dem Vokabular der _TEI Guidelines_ ist die ausführliche Dokumentation mit den zahlreichen darin enthaltenen Beispielen. Um den Einstieg zu erleichtern, sind in den einzelnen Kapiteln dieses Curriculums regelmäßig Links zu den entsprechenden TEI-Modulen angegeben.

> *Tipp:* Öffnen Sie die [Guidelines](https://tei-c.org/guidelines/P5/) während Ihrer Arbeit. Dort finden Sie Links zu Listen mit [Elementen](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/REF-ELEMENTS.html) und [Attributen](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/REF-ATTS.html), oder Sie können direkt nach einzelnen Komponenten [suchen](https://tei-c.org/release/doc/tei-p5-doc/en/html/index.html).

Reviewer: ```suggestion
*Tipp:* Öffnen Sie die _[Guidelines](https://tei-c.org/guidelines/P5/)_ während Ihrer Arbeit. Dort finden Sie Links zu Listen mit [Elementen](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/REF-ELEMENTS.html) und [Attributen](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/REF-ATTS.html), oder Sie können direkt nach einzelnen Komponenten [suchen](https://tei-c.org/release/doc/tei-p5-doc/en/html/index.html).
```
=> Nur kurisviert.

---

> title: 2. Einführung TEI Übungsaufgabe

Reviewer 2: Ich würde hier nicht nochmal "Einführung" nehmen, nach "Einführung in die Markup Sprachen.
```suggestion
title: 2. (Einleitende) Erklärung der TEI Übungsaufgabe
```

> title: 2. Einführung TEI Übungsaufgabe

Reviewer: ```suggestion
title: 2. Einführung in die TEI-Annotation
```

lang: de
date: 2021-07-12T07:02:59.186Z
version: 1.0.0
authors:
  - czeitschner-ulrike
  - schopper-daniel
editors:
  - königshofer-elisabeth
  - wiencek-florian
tags:
  - xml
  - tei

> abstract: Überblick über die TEI Übungsbeispiele des Curriculums.

Reviewer: ```suggestion
abstract: Überblick über die Beispieltexte des Curriculums.
```

licence: ccby-4.0
toc: false
uuid: R7SuQIsERiv1eweyKmmcq
---

> ## Über das Übungsbeispiel

Reviewer 2: Hier stellt sich die Frage, ob man nur das eine Übungsbeispiel hat oder mehrere, und diese dann auch "einleitende Worte" erhalten; oder man definiert es in der Überschrift: Über das Übungsbeispiel _Die Filmwelt_

Reviewer 3: ich würd hier bei einem Beispiel bleiben, das in die Überschrift nehmen. Zumindest solange wir kein weiteres Übungsbeispiel haben...

> ## Über das Übungsbeispiel

Reviewer: ```suggestion
## Über die Textbeispiele
```

Reviewer 3: Stimme zu, das ist nicht notwendig.

> Die Übungsbeispiele sollen Sie durch diese Einführung in TEI begleiten und Ihnen beispielhaft einige Möglichkeiten aufzeigen, wofür die Guidelines der Text Encoding Initiative genutzt werden können.

Reviewer 2: Übungsbeispiele - hier Plural:  ist es jetzt ein Übungstext und mehrere praktische Anwendungen, oder mehrere Beispieltexte?

Reviewer 3: siehe oben

> Die Übungsbeispiele sollen Sie durch diese Einführung in TEI begleiten und Ihnen beispielhaft einige Möglichkeiten aufzeigen, wofür die Guidelines der Text Encoding Initiative genutzt werden können.

Reviewer: ```suggestion
Die gewählten Textbeispiele sollen Sie durch dieses begleiten und Ihnen beispielhaft einige Möglichkeiten aufzeigen, wofür die Guidelines der Text Encoding Initiative genutzt werden können.
```

> Kodiert werden die Seiten 11 und 12 aus der in Wien erschienenen Zeitschrift Die Filmwelt. Nr. 31, Jahrgang 1924. Der gewählte Ausschnitt umfasst eine kurze Filmbesprechung inklusive Spielplan, ein Gedicht, mehrere kurze Beiträge aus der Rubrik 'Film-Kritiken' sowie eine Bekanntmachung der Zeitschriftenredaktion. Diese Zeitschrift ist online verfügbar unter [ÖNB-ANNO](http://anno.onb.ac.at/) ([Filmwelt 31, 1924, Seite 11](http://anno.onb.ac.at/cgi-content/annoshow-plus?call=fwt|1924|0131|00000011||jpg||45|), [Filmwelt 31, 1924, Seite 12](http://anno.onb.ac.at/cgi-content/annoshow-plus?call=fwt|1924|0131|00000012||jpg||45|)).

Reviewer 2: ```suggestion
Kodiert werden die Seiten 11 und 12 aus der in Wien erschienenen Zeitschrift _Die Filmwelt_, Nr. 31, Jahrgang 1924. Der gewählte Ausschnitt umfasst eine kurze Filmbesprechung inklusive Spielplan, ein Gedicht, mehrere kurze Beiträge aus der Rubrik "Film-Kritiken" sowie eine Bekanntmachung der Zeitschriftenredaktion. Diese Zeitschrift ist online verfügbar unter [ÖNB-ANNO](http://anno.onb.ac.at/) ([Filmwelt 31, 1924, Seite 11](http://anno.onb.ac.at/cgi-content/annoshow-plus?call=fwt|1924|0131|00000011||jpg||45|), [Filmwelt 31, 1924, Seite 12](http://anno.onb.ac.at/cgi-content/annoshow-plus?call=fwt|1924|0131|00000012||jpg||45|)).
```

> Kodiert werden die Seiten 11 und 12 aus der in Wien erschienenen Zeitschrift Die Filmwelt. Nr. 31, Jahrgang 1924. Der gewählte Ausschnitt umfasst eine kurze Filmbesprechung inklusive Spielplan, ein Gedicht, mehrere kurze Beiträge aus der Rubrik 'Film-Kritiken' sowie eine Bekanntmachung der Zeitschriftenredaktion. Diese Zeitschrift ist online verfügbar unter [ÖNB-ANNO](http://anno.onb.ac.at/) ([Filmwelt 31, 1924, Seite 11](http://anno.onb.ac.at/cgi-content/annoshow-plus?call=fwt|1924|0131|00000011||jpg||45|), [Filmwelt 31, 1924, Seite 12](http://anno.onb.ac.at/cgi-content/annoshow-plus?call=fwt|1924|0131|00000012||jpg||45|)).

Reviewer: ```suggestion
Kodiert werden die Seiten 11 und 12 aus der in Wien erschienenen Zeitschrift _Die Filmwelt. Nr. 31, Jahrgang 1924._ Der gewählte Ausschnitt umfasst eine kurze Filmbesprechung inklusive Spielplan, ein Gedicht, mehrere kurze Beiträge aus der Rubrik "Film-Kritiken" sowie eine Bekanntmachung der Zeitschriftenredaktion. Diese Zeitschrift ist online verfügbar unter [ÖNB-ANNO](http://anno.onb.ac.at/) ([Filmwelt 31, 1924, Seite 11](http://anno.onb.ac.at/cgi-content/annoshow-plus?call=fwt|1924|0131|00000011||jpg||45|), [Filmwelt 31, 1924, Seite 12](http://anno.onb.ac.at/cgi-content/annoshow-plus?call=fwt|1924|0131|00000012||jpg||45|)).
```

Reviewer: => Die Kursivierung wurde hier nicht übernommen, wieso?

![Filmwelt 31, 1924, S. 11, mit Bereichsannotation](images/filmwelt_1923_31_11_annotiert.jpg)

![Filmwelt 31, 1924, S. 12, mit Bereichsannotation](images/filmwelt_1923_31_12_annotiert.jpg)

> Anhand dieses Beispiels wollen wir gemeinsam die unterschiedlichen Facetten erarbeiten, die in einem solchen Dokument kodiert werden können. Hierfür gehen wir von beispielhaften Forschungsfragen aus, die man an das Dokument stellen könnte und die sich durch die vorgestellte Kodierung beantworten lassen.

Reviewer 2: ```suggestion
Anhand dieses Beispiels wollen wir gemeinsam die unterschiedlichen Facetten erarbeiten, die in einem solchen Dokument kodiert werden können. Hierfür gehen wir von beispielhaften Forschungsfragen aus, die man an den Text stellen könnte und die sich durch die vorgestellte Kodierung beantworten lassen.
```

> Anhand dieses Beispiels wollen wir gemeinsam die unterschiedlichen Facetten erarbeiten, die in einem solchen Dokument kodiert werden können. Hierfür gehen wir von beispielhaften Forschungsfragen aus, die man an das Dokument stellen könnte und die sich durch die vorgestellte Kodierung beantworten lassen.

Reviewer: => Blabla

> Anhand dieses Beispiels wollen wir gemeinsam die unterschiedlichen Facetten erarbeiten, die in einem solchen Dokument kodiert werden können. Hierfür gehen wir von beispielhaften Forschungsfragen aus, die man an das Dokument stellen könnte und die sich durch die vorgestellte Kodierung beantworten lassen.

Reviewer: => überflüssig

> Anhand dieses Beispiels wollen wir gemeinsam die unterschiedlichen Facetten erarbeiten, die in einem solchen Dokument kodiert werden können. Hierfür gehen wir von beispielhaften Forschungsfragen aus, die man an das Dokument stellen könnte und die sich durch die vorgestellte Kodierung beantworten lassen.

Reviewer: ```suggestion
Anhand dieser Vorlage werden unterschiedliche Facetten der Textannotation vorgestellt.
```
=> _Hierfür gehen wir von beispielhaften Forschungsfragen aus, die man an das Dokument stellen könnte und die sich durch die vorgestellte Kodierung beantworten lassen._ Überflüssig ... bzw. hängt davon ab, ob tatsächlich "Forschungsfragen" formuliert wurden ...

Reviewer 3: Müsste man schauen, ob man die Formulierungen, die wir gewählt haben (siehe spätere Module), als Forschungsfragen durchgehen lassen kann. Das würde ich noch im Raum stehen lassen. Den Rest der Korrektur bitte übernehmen.

Reviewer 2: Die textliche Änderung ist übernommen und scheint auch so im CMS bzw. im "Files changed" auf; das Kommentar habe ich nochmal aufgemacht, damit wir dann den Punkt mit den Forschungsfragen klären.

> Natürlich können wir in diesem Use-Case nicht jedes Detail der TEI Spezifikation behandeln. Ein wichtiges Hilfsmittel für die Arbeit mit dem Vokabular der TEI Guidelines ist daher die ausführliche Dokumentation mit den zahlreichen darin enthaltenen Beispielen. Die relevanten Passagen aus den Guidelines werden in der Einführung zusätzlich verlinkt.

Reviewer: ```suggestion
Ein wichtiges Hilfsmittel für die Arbeit mit dem Vokabular der _TEI Guidelines_ ist die ausführliche Dokumentation mit den zahlreichen darin enthaltenen Beispielen. Um den Einstieg zu erleichtern, wurde zu den für dieses Curriculum relevanten Passagen verlinkt.
```

> *Tip:* Öffnen Sie die [Guidelines](https://tei-c.org/guidelines/P5/) während Ihrer Arbeit. Dort finden Sie Links zu Listen mit [Elementen](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/REF-ELEMENTS.html) und [Attributen](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/REF-ATTS.html), oder Sie können direkt nach einzelnen Komponenten suchen.

Reviewer 2: ```suggestion
*Tipp:* Öffnen Sie die [Guidelines](https://tei-c.org/guidelines/P5/) während Ihrer Arbeit. Dort finden Sie Links zu Listen mit [Elementen](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/REF-ELEMENTS.html) und [Attributen](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/REF-ATTS.html), oder Sie können direkt nach einzelnen Komponenten suchen.
```

> *Tip:* Öffnen Sie die [Guidelines](https://tei-c.org/guidelines/P5/) während Ihrer Arbeit. Dort finden Sie Links zu Listen mit [Elementen](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/REF-ELEMENTS.html) und [Attributen](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/REF-ATTS.html), oder Sie können direkt nach einzelnen Komponenten suchen.

Reviewer: ```suggestion
*Tipp:* Ein wichtiges Hilfsmittel für die Arbeit mit dem Vokabular der _TEI Guidelines_ ist die ausführliche Dokumentation mit den zahlreichen darin enthaltenen Beispielen. Öffnen Sie die [Guidelines](https://tei-c.org/guidelines/P5/) während Ihrer Arbeit. Dort finden Sie Links zu Listen mit [Elementen](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/REF-ELEMENTS.html) und [Attributen](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/REF-ATTS.html), oder Sie können direkt nach einzelnen Komponenten suchen.
```

> *Tip:* Öffnen Sie die [Guidelines](https://tei-c.org/guidelines/P5/) während Ihrer Arbeit. Dort finden Sie Links zu Listen mit [Elementen](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/REF-ELEMENTS.html) und [Attributen](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/REF-ATTS.html), oder Sie können direkt nach einzelnen Komponenten suchen.

Reviewer: ```suggestion
*Tipp:* Öffnen Sie die [Guidelines](https://tei-c.org/guidelines/P5/) während Ihrer Arbeit. Dort finden Sie Links zu Listen mit [Elementen](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/REF-ELEMENTS.html) und [Attributen](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/REF-ATTS.html), oder Sie können direkt nach einzelnen Komponenten suchen.
```

> Das Ziel der Übung sollte es sein, dass Sie in jedem Kapitel zumindest einige Belegstellen selbst annotieren.

Reviewer: => Kann ich erst später beurteilen, ob "Übung" und was das Ziel hier ist ... "einige Fragen beantwnrten?" ...

> Das Ziel der Übung sollte es sein, dass Sie in jeder Praxis-Einheit zumindest einige Belegstellen selbst annotieren.

Reviewer 2: Das Wort "Praxis-Einheit" müsste man ggf. noch an das How-To angleichen, da hier die "praktischen Anwendungen" auf kleine quizzes aufteilt. 
e.g. Ziel ist es, dass Sie in jedem Kapitel/in jeder praktischen Anwendung...

Reviewer 3: eher Kapitel, da ja einer unserer Unterüberschriften innerhalb des Kapitels "praktische Anwendung" ist.

> Das Ziel dieses Curriculums ist es, Ihnen die Grundzüge der TEI-Annotation näher zu bringen und Sie damit in die Lage zu versetzen, die hier vorbereiteten "Wissenstests" und "Übungen" mühelos zu absolvieren. 

Reviewer: ```suggestion
Das Ziel dieses Curriculums ist es, Ihnen die Grundzüge der TEI-Annotation näher zu bringen und Sie damit in die Lage zu versetzen, die hier vorbereiteten Aufgaben mühelos zu absolvieren. 
```
